manifests/r/Rhinode/TradingPaints/2.0.35/Rhinode.TradingPaints.installer.yaml

```
@@ -0,0 +1,24 @@
+﻿# Created using YamlCreate.ps1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.0.0.schema.json
+
+PackageIdentifier: Rhinode.TradingPaints
+PackageVersion: 2.0.35
+MinimumOSVersion: 10.0.0.0
+InstallModes:
+- interactive
+- silent
+- silentWithProgress
+Installers:
+- InstallerLocale: en-US
+  Architecture: x86
```

```suggestion
  Architecture: x64
```
Lets try this then.. I believe the x86 machine is scuffed cuz its often causing manifests to fail and then not throwing a proper error..